website/docs/r/cce_node_pool.html.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: "huaweicloud"
+page_title: "huaweicloud: huaweicloud_cce_node_pool"
+sidebar_current: "docs-huaweicloud-resource-cce-node-pool"
+description: |-
+  Add a node pool to a container cluster. 
+---
+
+# huaweicloud\_cce\_node\_pool
+Add a node pool to a container cluster. 
+This is an alternative to `huaweicloud_cce_node_pool`
+
+## Example Usage
+
+```hcl
+variable "cluster_id" { }
+variable "key_pair" { }
+variable "availability_zone" { }
+
+resource "huaweicloud_cce_node_pool" "node_pool" {
+  cluster_id        = var.cluster_id
+  name              = "testpool"
+  os                = "EulerOS"
+  initial_node_count = 2
+  flavor_id         = "s3.large.4"
+  availability_zone = var.availability_zone
+  key_pair          = var.keypair
+  region           =  "cn-east-3"
+  scall_enable      = true
+  min_node_count    = 1
+  max_node_count    = 10
+  scale_down_cooldown_time = 100
+  priority         = 1
+
+  root_volume {
+    size       = 40
+    volumetype = "SAS"
+  }
+  data_volumes {
+    size       = 100
+    volumetype = "SAS"
+  }
+}
+``` 
+
+## Argument Reference
+The following arguments are supported:
+
+* `cluster_id` - (Required) ID of the cluster. Changing this parameter will create a new resource.
+
+* `name` - (Required) Node Pool Name.
+
+* `initial_node_count` - (Required) Initial number of expected nodes in the node pool.
+
+* `flavor_id` - (Required) Specifies the flavor id. Changing this parameter will create a new resource.
+ 
+* `availability_zone` - (Required) specify the name of the available partition (AZ). Changing this parameter will create a new resource.
+
+* `os` - (Optional) Operating System of the node. Changing this parameter will create a new resource.
+    - For VM nodes, clusters of v1.13 and later support *EulerOS 2.5* and *CentOS 7.6*.
+    - For BMS nodes purchased in the yearly/monthly billing mode, only *EulerOS 2.3* is supported.
```

Comment:
node pools only support VM nodes cluster, so the supported OS are  *EulerOS 2.5* and *CentOS 7.6*